src/traits/trait-objects.md

```diff
@@ -47,32 +39,40 @@ Memory layout after allocating `pets`:
 :   +-----------+-------+   :     :   +-----+-----+                             :
 :   | ptr       |   o---+---+-----+-->| o o | o o |                             :
 :   | len       |     2 |   :     :   +-|-|-+-|-|-+                             :
-:   | capacity  |     2 |   :     :     | |   | |   +---------------+           :
-:   +-----------+-------+   :     :     | |   | '-->| name: "Fido"  |           :
-:                           :     :     | |   |     +---------------+           :
+:   | capacity  |     2 |   :     :     | |   | |   +-------------+             :
+:   +-----------+-------+   :     :     | |   | '-->| name: Fido  |             :
```

Comment:
Maybe `{name: "Fido"}` just to suggest struct-ness and string-ness a bit more?

Reply:
I see what you mean — though the style on the right is consistent with how the `Vec` fields are shown on the left. I've used this "table style" throughout.

Now, I did try to make Fido be `"Fido"`, but I didn't manage to escape the `"` for the svgbob plugin. The double quote is used to quote special characters for the plugin.

Reply:
I think I have a nicer and more correct solution: I now show the `"Fido"` string as a separate object. This is consistent with that we do in the diagrams for moved strings.